include/mbedtls/ssl.h

```diff
@@ -2969,6 +2972,13 @@ void mbedtls_ssl_conf_curves( mbedtls_ssl_config *conf,
  * \note           This list should be ordered by decreasing preference
  *                 (preferred hash first).
  *
+ * \note           By default, all supported hashes whose length is at least
+ *                 256 bits are allowed. This is the same set as the default
+ *                 for certificate verification
+ *                 (#mbedtls_x509_crt_profile_default).
+ *                 The preference order is currently unspecified and may
+ *                 change in future versions.
```

Comment:
We may or may not want change the documentation of `mbedtls_md_list`, which currently guarantees that the list “starts with the strongest available hashes” — which by the way means that the list has to be ordered from strongest to weakest, since disabling any subset must leave the strongest remaining hash at the top.

Related: [where to add SM3 in `mbedtls_md_list()`](https://github.com/ARMmbed/mbedtls/pull/4492#issuecomment-853779085)